sys/include/ztimer.h

```diff
@@ -301,13 +301,26 @@ struct ztimer_base {
     uint32_t offset;            /**< offset from last timer in list */
 };
 
+/**
+ * @defgroup    ztimer_now64 ztimer_now64
+ * @deprecated  use uint32_t or ztimer64, will be removed after 2022.07 release.
+ *
+ * @brief deprecated module changes deprecated ztimer_now_t type
+ *
+ * use ztimer_now() returning uint32_t or alternatively use module ztimer64
+ * ztimer64_now() returning uint64_t.
+ * @{
+ */
 #if MODULE_ZTIMER_NOW64
 typedef uint64_t ztimer_now_t;  /**< type for ztimer_now() result
-                                 * @deprecated use uint32_t or ztimer64 */
+                                 * @deprecated use uint32_t or ztimer64,
+                                 * will be removed after 2022.07 release. */
 #else
 typedef uint32_t ztimer_now_t;  /**< type for ztimer_now() result
```

Comment:
Move `@brief` above and update deprecation note to be more understandable, as suggested above.

```suggestion
/**
 * @brief type for ztimer_now() result
 * @deprecated use @ref ztimer_now() returning uint32_t or alternatively use module
 *             `ztimer64` with @ref ztimer64_now() returning uint64_t.
 *             Will be removed after 2022.10 release.
 */
typedef uint64_t ztimer_now_t;
```